aretomo/protocols/protocol_aretomo.py

```diff
@@ -376,6 +395,7 @@ def convertInputStep(self, tsObjId: int, tsId: str, tsFn: str):
             else:
                 raise FileNotFoundError("Missing input aln file ", protAlnBase)
 
+    @tryExceptDecorator
```

Comment:
Shouldn't we add the decorator to the createOutputStep? as well?

Reply:
It seems logical to me to add it only to the program execution step, as we're trying to capture the ones the program fails with. But I get the point, it may happen that the program does not fail and the output files are not generated properly or something that makes the createOutputStep fail. I haven't found this kind of cases. @azazellochg?

Reply:
I havent either